crates/net/eth-wire-types/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+//! Types for the eth wire protocol.
```

Comment:
this needs the regular header

like this:

https://github.com/paradigmxyz/reth/blob/33c466d88c4bee67f0b4eb15b2ca1b2fc421b10b/crates/net/nat/src/lib.rs#L7-L12